en/06_Texture_mapping/00_Images.adoc

```diff
@@ -59,12 +59,12 @@ Add the directory with `stb_image.h` to the include directories for GCC:
 
 [,text]
 ----
-VULKAN_SDK_PATH = /home/user/VulkanSDK/x.x.x.x/x86_64
+VULKAN_SDK_PATH = /usr/include/vulkan
```

Comment:
This might not be correct.  I'd have to look at the instructions again if LunarG advocates to install it to the system include directory.  Having the includes live in the VulkanSDK path is universally accurate and covers the case where a linux user might not have super user access.